src/main/java/org/jahia/modules/serversettings/flow/WebprojectHandler.java

```diff
@@ -385,8 +387,14 @@ public void exportToFile(RequestContext requestContext, boolean staging) throws
                 siteList.add((JCRSiteNode) site);
             }
         }
-
-        importExportBaseService.exportSites(new ByteArrayOutputStream(), params, siteList);
+        try {
+            importExportBaseService.exportSites(new ByteArrayOutputStream(), params, siteList);
+        } catch (JahiaForbiddenAccessException e) {
+            logger.error(e.getMessage());
+            requestContext.getMessageContext().addMessage(new MessageBuilder().error()
+                .code("serverSettings.manageWebProjects.exportPath.invalidEmptyFolder")
```

Comment:
You set a specific message "Unable to export site because the folder provided is not empty" for an exception, which can have multiple reasons. So the error is misleading and unclear as it is not always that exact reason.

For instance if I try to set ../../myfolder - this folder is not existing (or empty) - but here the problem is that I am not allowed to specify a path outside of the configured export location.